core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -932,6 +927,7 @@ class LogManager(logDirs: Seq[File],
         val logsToCheckpoint = logsInDir(logDir)
         checkpointRecoveryOffsetsAndCleanSnapshotsInDir(logDir, logsToCheckpoint, ArrayBuffer.empty)
         checkpointLogStartOffsetsInDir(logDir, logsToCheckpoint)
+        sourceLog.removeLogMetrics()
```

Comment:
This seems a bit odd but looks like an existing issue. We are removing metrics for the `current` log but we leave the metrics for the `future` log as is, which means it will retain the `is-future` tag. Maybe worth filing a JIRA if you think it is a problem too.

Reply:
Good catch. Opened https://issues.apache.org/jira/browse/KAFKA-10320 to track it.